src/App.tsx

```diff
@@ -1,24 +1,263 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
 
-const USER_ID = 0;
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+import { Header } from './components/Header';
+import { ErrorNotification } from './components/ErrorNotification';
+import { TodoItem } from './components/TodoItem';
+import * as todosService from './api/todos';
+import { USER_ID } from './api/todos';
+import * as filterService from './utils/filterService';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [loadingTodos, setLoadingTodos] = useState(false);
```

Comment:
Loading is an ongoing process, if you want to get info about whether todos were loaded - then you need to choose 'isTodoLoaded' or stuff like this. It can confuse other developers in your team.